### DISCUSSION TITLE
How can I auto-populate a dropdown/submenu list based on tags?

### DISCUSSION THREAD

**Discussion by dixonge** [GetPublii/Publii#1771] [Help]
I would like to be able to hover over a menu item and have all posts with a specific tag show up in the dropdown menu. I would rather not manually add all new posts to the menu, and I would rather not have a main menu item just open a tag page.

Is this possible in Publii? (see screenshot for rendering in Grav CMS)

![SCR-20241024-fzns](https://github.com/user-attachments/assets/223059f7-8958-4428-9f2e-339b45c6c442)

**Comment by bjazmoore** [comment]
That is an interesting idea.  Grav is a PHP based blog tool and can do this on the fly.  It might be possible for the development team to add this to the core of Publii using JavaScript.  It does not do this now.
